scripts/linux/fv/olp-cpp-sdk-integration-test.variables

```diff
@@ -0,0 +1,6 @@
+export dataservice_read_test_secret="${dataservice_read_test_secret}" #defined previously as protected value on Gitlab
```

Comment:
exporting an environment variable from itself? Does this even work?

Reply:
This is approach we use on FV to exclude hardcoded tokens from scripts. Those variables are setup on Gitlab Project Settings Page.

Reply:
Yes, it works.  I want all valiables to be in the same place as it was before. Having 2 different names for variable that represent one entity is more confusing than this.

Reply:
To be sure we understand each other. You are exporting the same environment variable again as string. That was the reason why it had a different name. It is not very nice that you override env variables passed down from above.

Reply:
>  exporting the same environment variable again as string.
I'm not sure what you mean. Environment variables are strings, as well as any other shell variable.

Reply:
Ok, let me talk trough code
```bash
# Envirnoment variable set by GitLab CI
export dataservice_read_test_secret = "token_sercret_1"

# Your changes in scripts/linux/fv/olp-cpp-sdk-integration-test.variables
# which export the same environment variable again.
export dataservice_read_test_secret = {$dataservice_read_test_secret}
```
Why?

Reply:
Because this way all variables are visible to the shell in one place and in case it will ever change no need to look for them across whole codebase on multiple CI environments.

Reply:
Seems more like 3 places, 3 scripts, not one. And it is only one CI environment, gitlab. A comment would suffice instead of reexporting them again

Reply:
For every test there is one .variables file that explicitly lists all required variables. These files are visible to both GitLab and Travis CI. ```export``` is as cheap as variable assignment. Why do you think it is better to remove explicit code in favor of comments that people have tendency to not read?

Reply:
Ok, lets park the discussion for now.
I would propose in the future to focus on the ticket itself instead of refactore every piece of code on the way, as this is an endless loop.